dev/example_dags/README.md

```diff
@@ -1,34 +0,0 @@
-<!--
```

Comment:
This had not been used any more for refreshing old example_dags path. We can just remove those.